### PR TITLE
Added anchor links to headers in the docs

### PR DESCRIPTION
**Pull request by tobilg**
* Added [anchor.js](https://github.com/bryanbraun/anchorjs) to dynamically render the link icons
* Added according CSS

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -58,5 +59,13 @@
     </div>
     <script src="//cdn.jsdelivr.net/jquery/2.1.1/jquery.min.js"></script>
     <script src="//cdn.jsdelivr.net/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.2/anchor.min.js"></script>
```
Comment (janisz) [inline docs/_layouts/default.html]: Other links comes form jsdeliver why you choose cloudflare? https://cdn.jsdelivr.net/anchorjs/3.2.2/anchor.min.js

**Comment by janisz** [comment]
@tobilg Nice! I'd love to see anchors in Mesos docs too.

File: docs/_layouts/default.html (hunk 1 above)
Comment (tobilg) [inline docs/_layouts/default.html]: I was sticking to the project's docs. Changed it now.

**Comment by tobilg** [comment]
Are there any blockers for getting this merged? Thanks!

**Comment by jdef** [comment]
@sascala can you confirm that this is compatible with any other in-house documentation generation process/tooling that we're using?

**Comment by sascala** [comment]
LGTM on a local build

**Pull request merged**